playbooks/openshift/roles/deploy/tasks/set_docker_registry.yml

```diff
@@ -0,0 +1,22 @@
+- name: comment out current registry lines
```

Comment:
I also do not use this playbook, as you can see from https://github.com/herlo/paas-sig-ci/blob/0d97814baf30873961f5bff56ee81e4186525aee/playbooks/openshift/roles/deploy/tasks/main.yml#L3.

I'll remove this file shortly.